content/blog/2016/2016-04-07-pipeline-for-runs-on-hardware.adoc

```diff
@@ -0,0 +1,258 @@
+---
+layout: post
+title: Automating test runs on hardware with Pipeline-as-Code. Part 1
+tags:
+- jenkins
+- jenkins2
+- pipeline
+- embedded
+author: oleg_nenashev
+---
+
+In addition to Jenkins development, during last 8 years I've been involved into continuous integration for hardware and embedded projects.
+At JUC2015/London 
+link:https://www.cloudbees.com/jenkins/juc-2015/presentations/JUC-2015-Europe-Jenkins-Based-CI-for-Nenashev.pdf[I have conducted a talk] about common automation challenges in the area.
+
+In this blog post I would like to concentrate on link:https://jenkins.io/doc/pipeline/[Pipeline] (fka Workflow), which is a new automation-as-code ecosystem for Jenkins.
+It will be proposed by default in the incoming link:/2.0/[Jenkins 2.0 release].
```

Comment:
This statement's tense is incorrect, it is not "being proposed to be included by default" so to say, but rather it's included in the suggested plugins list in 2.0.

I suggest something along the lines of "It is in the suggested plugins list in the upcoming Jenkins 2.0 release"